package.json

```diff
@@ -49,9 +49,12 @@
     "jsdom": "^26.0.0",
     "lint-staged": "^15.2.9",
     "prettier": "^3.4.2",
-    "typescript": "^4.7.4",
+    "typescript": "^4.9.5",
     "vitest": "^3.0.7"
   },
+  "overrides": {
+    "typescript": "^4.9.5"
```

Comment:
This is needed to resolve installed typescript properly. Typedoc dependency pulls another one